src/api/libcellml/parentedentity.h

```diff
@@ -83,7 +76,6 @@ class LIBCELLML_EXPORT ParentedEntity: public Entity
     explicit ParentedEntity(ParentedEntityImpl *pImpl); /**< Constructor for derived classes, @private. */
 
 private:
-    ParentedEntityImpl *pFunc(); /**< Getter for private implementation pointer, @private. */
```

Comment:
Why remove it?

Reply:
Because `ParentedEntity` only uses the const form of pFunc.

Reply:
I thought it might be the case and I was going to say that it doesn't harm having both signatures, but this would obvisouly break our coverage tests. Ok, fair enough then.